solutions/devicesimulation-vmss-nohub/single-vm/scripts/logs.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
All the files in scripts folder is duplicated across the device simulation folders. Can we move it to common folders and consume from there?

Reply:
scripts might differ depending on resources, I'd keep things separate